### PR TITLE
Preferring Single line comment over a multiline comment

### PR DESCRIPTION
**Pull request by Anurag-0-1-A**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
If there are any UI change, please include the screenshots also.

Fixes #(issue)

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [ ] `./gradlew assembleDebug assembleRelease`
- [ ] `./gradlew checkstyle`

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java [hunk 1]
```diff
@@ -108,9 +108,9 @@ public void onPageScrollStateChanged(int arg0) {
     };
 
 
-    /**
-     * View pager adapter
-     */
+    
+     // View pager adapter
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java:112]: why?

Comment (Anurag-0-1-A) [reply app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java:112]: There are no multiple lines here to use /*....*/ so preferring a single line comment here as observed the same above in the code.

Comment (Swati4star) [reply app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java:112]: Even if it is single line, we prefer javadoc style comments: http://drjava.org/docs/user/ch10.html

Comment (Anurag-0-1-A) [reply app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java:112]: I mean /* .. */ in my above comment.

Comment (Swati4star) [reply app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java:112]: If you wish to make the formatting uniform, please do it for whole file.
and don't make /* .. */  to //
We prefer javadoc style comments structure (* .. */ )

Comment (Anurag-0-1-A) [reply app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java:112]: Sure

**Pull request closed without merging**